flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -687,11 +687,11 @@ case class LogicalTableFunctionCall(
     parameters: Seq[Expression],
     resultType: TypeInformation[_],
     fieldNames: Array[String],
-    child: LogicalNode)
+    child: LogicalNode,
+    evalMethod: Option[Method])
```

Comment:
You can make the evalMethod default as None, so you do not need to change every call.